src/java.base/share/classes/java/nio/channels/Channels.java

```diff
@@ -259,14 +259,14 @@ public void close() throws IOException {
     /**
      * Constructs a channel that reads bytes from the given stream.
      *
-     * <p> The resulting channel will not be buffered; it will simply redirect
-     * its I/O operations to the given stream.  Closing the channel will in
-     * turn cause the stream to be closed.  </p>
+     * <p> The resulting channel will be in blocking mode and will not be
+     * buffered; it will simply redirect its I/O operations to the given stream.
+     * Closing the channel will in turn cause the stream to be closed.  </p>
```

Comment:
ReadableByteChannel doesn't have any notion of blocking mode so might be confusing to say that it returns a channel in blocking mode. Maybe it would be simpler to just insert a sentence into the paragraph to say that reading from the resulting channel will read (and therefore might blocking reading) from the input stream?

Reply:
Changed as suggested in dba2d33.